.travis.yml

```diff
@@ -5,8 +5,12 @@ services:
 cache:
   directories:
   - .glide
-script:
-- make verify build test images
+jobs:
+  include:
+    - stage: Verify and Build
+      script: make verify build
+    - stage: Test and Make Image
+      script: make test images
```

Comment:
@duglin If I make each step in different stage then each time a new container will be created and dependencies will be pulled in which will take lot of time and another 3-4 minutes in Build Time, still if the build time does not matter then I can split each step in different stages